include/kernels/coarse_op_kernel.cuh

```diff
@@ -220,36 +225,36 @@ namespace quda {
      @param[in] dim Dimension
      @param[in] arg Kernel argument
   */
-  template <typename Arg> constexpr bool isCoarseDiagonal(const int coord[], const int coord_coarse[], int dim, const Arg &arg)
+  template <typename Arg> constexpr bool isCoarseDiagonal(const int coord[], const int coord_coarse[], int dim, const int nFace, const Arg &arg)
```

Comment:
Not sure what the point is for this `const` qualifier for passing an integer (`nFace`) by value.

Reply:
https://github.com/lattice/quda/pull/1221/commits/494c0be0466700edb2329aeac14ced9a5ea7e9dd